### PR TITLE
Fix PackageName for all flavors

### LINKED ISSUE #1558: Rate Susi.AI link not found
**Actual Behaviour**

Item not found in google play store for rating susi.ai

Edit: The error was due to building the fdroid version, working fine for playstore version. But this won't work for fdroid Flavor

**Expected Behaviour**

It should open the app on playstore

**Steps to reproduce it**

Go to settings activity and then click on rate susi

**Would you like to work on the issue?**

Yes!

### PR DESCRIPTION
**Pull request by codedsun**
Fixes #1558 : Fix Package Name for All flavors

Changes: Added a new string with package name

@batbrain7 @arundhati24 Check!

### CODE REVIEW

File: app/src/main/java/org/fossasia/susi/ai/skills/settings/ChatSettingsFragment.kt [hunk 1]
```diff
@@ -62,6 +62,7 @@ class ChatSettingsFragment : PreferenceFragmentCompat(), ISettingsView {
     private lateinit var setupDevice: Preference
     private lateinit var settingsVoice: Preference
     private var flag = true
+    private var PACKAGENAME = "ai.susi";
```
Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/susi/ai/skills/settings/ChatSettingsFragment.kt]: Java variable naming guidelines

**Comment by codedsun** [comment]
@batbrain7 @arundhati24 Please check this!

File: app/src/main/java/org/fossasia/susi/ai/skills/settings/ChatSettingsFragment.kt [hunk 2]
```diff
@@ -62,6 +62,7 @@ class ChatSettingsFragment : PreferenceFragmentCompat(), ISettingsView {
     private lateinit var setupDevice: Preference
     private lateinit var settingsVoice: Preference
     private var flag = true
+    private var PACKAGE_NAME = "ai.susi";
```
Comment (batbrain7) [inline app/src/main/java/org/fossasia/susi/ai/skills/settings/ChatSettingsFragment.kt]: use val, also is camelcase not good?

**Comment by batbrain7** [comment]
@codedsun I may not know all the things that is why asked "is camel case not good?"

File: app/src/main/java/org/fossasia/susi/ai/skills/settings/ChatSettingsFragment.kt [hunk 3]
```diff
@@ -62,6 +62,7 @@ class ChatSettingsFragment : PreferenceFragmentCompat(), ISettingsView {
     private lateinit var setupDevice: Preference
     private lateinit var settingsVoice: Preference
     private var flag = true
+    private val packageName = "ai.susi";
```
Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/susi/ai/skills/settings/ChatSettingsFragment.kt]: BuildConfig.APPLICATION_ID

Comment (codedsun) [reply app/src/main/java/org/fossasia/susi/ai/skills/settings/ChatSettingsFragment.kt]: @iamareebjamal For fdroid flavor it becomes ai.susi.fdroid, which can't be opened on play store

Comment (codedsun) [reply app/src/main/java/org/fossasia/susi/ai/skills/settings/ChatSettingsFragment.kt]: That's why I have hardcoded it 👍

Comment (iamareebjamal) [reply app/src/main/java/org/fossasia/susi/ai/skills/settings/ChatSettingsFragment.kt]: Did you try getting it this way?

Comment (codedsun) [reply app/src/main/java/org/fossasia/susi/ai/skills/settings/ChatSettingsFragment.kt]: @iamareebjamal Yes, I did! check the build it has ```applicationIdSuffix ".fdroid"```

Comment (iamareebjamal) [reply app/src/main/java/org/fossasia/susi/ai/skills/settings/ChatSettingsFragment.kt]: That's fine, I know that. I wanted to know what `BuildConfig.APPLICATION_ID` returned

Comment (codedsun) [reply app/src/main/java/org/fossasia/susi/ai/skills/settings/ChatSettingsFragment.kt]: @iamareebjamal For fdroid version, it returned ai.susi.fdroid while for playstore version it returned ai.susi

**Comment by codedsun** [comment]
Done 👍

**Comment by iamareebjamal** [comment]
Still not following commit guidelines

**Comment by codedsun** [comment]
@arundhati24 @batbrain7 Please guide me what exactly I am missing, I couldnt' figure it out.

**Comment by batbrain7** [comment]
read about commit guidelines @codedsun

**Comment by batbrain7** [comment]
did you read about commit guidelines?

**Comment by codedsun** [comment]
@batbrain7 Yes, already did 👍 and have been following the same guidelines. If you could point where I am wrong, that's what I will be requiring to proceed. Thanks ;)

**Comment by iamareebjamal** [comment]
Umm, please go to the dev branch and see previous commits in which form they are

Also, read this https://chris.beams.io/posts/git-commit/

**Comment by codedsun** [comment]
@iamareebjamal Does that commit msg is good?

**Comment by iamareebjamal** [comment]
Check the merged commit message

Follow that format

**Comment by iamareebjamal** [comment]
Even the PR name was good

**Pull request merged**